js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,34 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="pattern"
+        labelIcon={
+          <HelpItem
+            helpText={t("targetContextAttributesHelp")}
+            fieldLabelId="clients:targetContextAttributes"
+          />
+        }
+        isRequired
```

Comment:
Should this really be a required field? A checkbox already has a value at all times.

Reply:
no, indeed, it was not required